### PR TITLE
refactor: search in platform adapter

### PR DESCRIPTION
**Pull request by mnavarroespinosa**
EX-6313

### CODE REVIEW

File: packages/x-adapter-platform/src/types/platform-adapter.types.ts [hunk 1]
```diff
@@ -6,15 +6,12 @@ import {
...
 } from '@empathyco/x-types';
-import { BaseRequest, SearchRequest, TaggingRequest } from './request.types';
-import {
-  NextQueriesResponse,
-  RelatedTagsResponse,
-  SearchResponse,
-  TopClickedResponse
-} from './response.types';
+import { BaseRequest, TaggingRequest } from './request.types';
+import { NextQueriesResponse, RelatedTagsResponse, TopClickedResponse } from './response.types';
```
Comment (tiborux) [inline packages/x-adapter-platform/src/types/platform-adapter.types.ts:14]: NextQueriesResponse  and TopClickedResponse should not be imported from here, right?

File: packages/x-adapter-platform/src/types/request.types.ts [hunk 2]
```diff
@@ -1,6 +1,3 @@
-import { Filter } from '@empathyco/x-types';
-import { Dictionary } from '@empathyco/x-utils';
-
 export interface BaseRequest {
```
Comment (tiborux) [inline packages/x-adapter-platform/src/types/request.types.ts:1]: should we move this to a base-request file?

**Review by tiborux — CHANGES_REQUESTED** [review]

File: packages/x-adapter-platform/src/types/platform-adapter.types.ts (hunk 1 above)
Comment (tiborux) [reply packages/x-adapter-platform/src/types/platform-adapter.types.ts:14]: I think that this change is made on the abraham's PR

Comment (mnavarroespinosa) [reply packages/x-adapter-platform/src/types/platform-adapter.types.ts:14]: Yes, you're right, it should not be imported from there and that change is made in Abraham's PR.

File: packages/x-adapter-platform/src/schemas/filters/simple-filter.schema.ts [hunk 3]
```diff
@@ -1,14 +1,12 @@
 import { createMutableSchema, Schema } from '@empathyco/x-adapter-next';
 import { SimpleFilter } from '@empathyco/x-types';
-import { PlatformFacetFilter } from '../../types';
+import { PlatformFilter } from '../../types/models/facet.model';
 
-export const simpleFilterSchema: Schema<PlatformFacetFilter, SimpleFilter> = {
+export const simpleMutableFilterSchema = createMutableSchema<Schema<PlatformFilter, SimpleFilter>>({
```
Comment (diegopf) [inline packages/x-adapter-platform/src/schemas/filters/simple-filter.schema.ts:5]:
```suggestion
export const simpleFilterMutableSchema = createMutableSchema<Schema<PlatformFilter, SimpleFilter>>({
```

File: packages/x-adapter-platform/src/schemas/models/facet.schema.ts [hunk 4]
```diff
@@ -7,14 +7,16 @@ import {
   NumberRangeFacet,
   SimpleFacet
 } from '@empathyco/x-types';
-import { PlatformFacet, PlatformHierarchicalFilter } from '../types/models.types';
-import { numberFilterMutableSchema } from './filters/number-filter.schema';
-import { simpleMutableFilterSchema } from './filters/simple-filter.schema';
+import { numberFilterMutableSchema } from '../filters/number-filter.schema';
+import { simpleMutableFilterSchema } from '../filters/simple-filter.schema';
```
Comment (diegopf) [inline packages/x-adapter-platform/src/schemas/models/facet.schema.ts:11]:
```suggestion
import { simpleFilterMutableSchema } from '../filters/simple-filter.schema';
```

File: packages/x-adapter-platform/src/types/request.types.ts (hunk 2 above)
Comment (diegopf) [reply packages/x-adapter-platform/src/types/request.types.ts:1]: will be used when we finish this refactor? I thought we are getting rid of this BaseRequest after all.

File: packages/x-adapter-platform/src/types/requests/index.ts [hunk 5]
```diff
@@ -1,3 +1,4 @@
-export * from './popular-searches-request.model';
-export * from './query-suggestions-request.model';
 export * from './identifier-results-request.model';
+export * from './query-suggestions-request.model';
+export * from './popular-searches-request.model';
+export * from './search-request.model';
```
Comment (diegopf) [inline packages/x-adapter-platform/src/types/requests/index.ts:4]: Next queries is missing here but I think that is solved on main already.

**Review by diegopf — CHANGES_REQUESTED** [review]

File: packages/x-adapter-platform/src/schemas/models/facet.schema.ts (hunk 4 above)
Comment (mnavarroespinosa) [reply packages/x-adapter-platform/src/schemas/models/facet.schema.ts]: Changed! Thanks!

File: packages/x-adapter-platform/src/types/requests/index.ts (hunk 5 above)
Comment (mnavarroespinosa) [reply packages/x-adapter-platform/src/types/requests/index.ts:6]: Updated with main and now Next queries are there.

**Review by tiborux — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**